toolkit/imageconfigs/distroless-minimal.json

```diff
@@ -0,0 +1,21 @@
+{
+    "Disks": [
+        {
+            "Artifacts": [
+                {
+                    "Name": "core",
```

Comment:
Could you change the name so there's no confusion with what's built from `core-container.json`?